faq.adoc

```diff
@@ -28,6 +28,15 @@ So far Kaitai Struct focuses on reading (parsing) only. There are plans to suppo
 
 == How does it compare to ...
 
+=== ... Python library Construct?
+
+Actually, Construct is the closest analog to Kaitai Struct. It is also a symmetric binary parsing library, but the difference is mostly that:
+
+* Construct does both parsing and serialization while Kaitai ([currently](https://github.com/kaitai-io/kaitai_struct/issues/27)) does only parsing
```

Comment:
We never call KS "Kaitai", especially in official documentation. Also, this is not Markdown, it's AsciiDoc, so this link won't work. It should be formatted like

```
https://github.com/kaitai-io/kaitai_struct/issues/27[currently]
```

Which, in turn, shows that you haven't even bothered to check before submitting.

Reply:
You are kinda wrong on this one. I didnt compile the docs because I dont know how. I did wonder how to express links, so looked in the remaining content and found this syntax so I used it. Sorry but I never worked with this format before.

Reply:
Building the docs is essentially a matter of running "make", once you'll get asciidoctor (or actually any other asciidoc processor of your choice) installed: https://github.com/kaitai-io/kaitai_struct_doc/blob/master/.travis.yml#L16-L20

Sorry about the mess with the other links, I'll try to fix them shortly.

Reply:
Maybe we could add a note in those "how to build compiler/run test" guide?

Reply:
Fixed.